static/app/views/settings/account/accountNotificationFineTuning.tsx

```diff
@@ -215,6 +215,16 @@ class AccountNotificationFineTuning extends DeprecatedAsyncView<Props, State> {
 
     const isProject = isGroupedByProject(fineTuneType) && organizations.length > 0;
     const field = ACCOUNT_NOTIFICATION_FIELDS[fineTuneType];
+    // TODO(isabella): once GA, remove this
+    if (
+      fineTuneType === 'quota' &&
+      organizations.some(org => org.features?.includes('spend-visibility-notifications'))
+    ) {
+      field.title = t('Spend Notifications');
+      field.description = t(
+        'Control the notifications you receive for organizations spend.'
```

Comment:
```suggestion
        "Control the notifications you receive for organization's spend."
```

I think this needs to be possessive.

Reply:
was following the copy on figma but this is a good point; since it's multiple organizations though maybe we should just do "organization spend", will bring up with the team

Reply:
'organization spend' would work too.